.github/ISSUE_TEMPLATE/03_documentation_bug.yml

```diff
@@ -0,0 +1,35 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Documentation bug
+description: >
+  Report a bug with the Carbon Language project documentation.
+labels: [explorer]
```

Comment:
What effect does this line have?

Reply:
Sets the label (incorrectly). Removing since we don't have a label for doc bugs (yet, at least). :)